modules/subnet/vars.tf

```diff
@@ -0,0 +1,25 @@
+variable "vpc_id"{
+    description = "vpc to deploy the subnet in"
+    default = "10.0.0.0/16"
```

Comment:
This is wrong if you say the `vpc_id` and you are passing a cidr notation as the default

Reply:
oooo.. Let me correct it

Reply:
I have fixed it. I will push the changes